protocol/0056-REWA-rewards_overview.md

```diff
@@ -184,6 +184,33 @@ $$m_{rz} = \frac{\sum_{i}^{n}{rz_{i}}}{N}$$
 
 Note, as a position can not be created on a Spot market. Trading activity on a Spot market will not contribute to this reward metric.
 
+
+### LP Performance Metric
+
+The LP performance metric, $m_{rz}$, measures the quality of the liquidity a party supplies.
+
+This metric differs from the [Liquidity Fees Received](#fee-based-reward-metrics) metric as instead of evaluating the actual liquidity fees received, the network evaluates how "virtual fees" would have been allocated and distributed. This is done to reward LPs for providing liquidity even in periods where this is reduced trading volume.
```

Comment:
```suggestion
This metric differs from the [Liquidity Fees Received](#fee-based-reward-metrics) metric as instead of evaluating the actual liquidity fees received, the network evaluates how "virtual fees" would have been allocated and distributed. This is done to reward LPs for providing liquidity even in periods where there is reduced trading volume.
```